exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/Function.kt

```diff
@@ -67,6 +67,15 @@ class Random(
 
 // String Functions
 
+/**
+ * Represents an SQL function that returns the length of [expr], measured in characters, or `null` if [expr] is null.
+ */
+class CharLength<T : String?>(
+    val expr: Expression<T>
```

Comment:
I would try to avoid shortcuts

Reply:
Shortcuts? Not sure what you mean, sorry. The formatting?

Reply:
`expr` -> `expression`